src/routes/FilterInterface.svelte

```diff
@@ -0,0 +1,71 @@
+<script lang="ts">
+
+	import { RecursiveTreeView, type TreeViewNode } from '@skeletonlabs/skeleton';
+  import { nodes } from '../treeContent';
+    
+let myTreeViewNodes: TreeViewNode[] = nodes;
+let checkedNodes : string[] = ['javascript'];
+let indeterminateNodes : string[] = ['programming', 'language'];
+											
+</script>
+
+<div class="dark-container w-screen flex pt-10 flex-col">
+  <h2 class ="h2 pl-16" style="text-align: left;" >Virtueller Katalog</h2>
+  <div class="side_by_side w-screen pl-2 pt-5 flex">
+    <div class="tree w-1/2 box-border text-sm">
+      
+      <RecursiveTreeView selection multiple relational {nodes} bind:checkedNodes={checkedNodes} bind:indeterminateNodes />
+      
+    </div>
+    <div class="search w-1/2 border-box">
+      <form class='form'>
+        <label class="searchView mr-10">
+            <input class="input text-black p-6 text-base placeholder-black" type="text" placeholder="Dies ist der Surchtext..." />
+        </label>
+        <span class="flex pl-6 pt-6 text-base">Suchen in</span>
+        <div class='categorySelect flex text-sm pl-5'>  
+            <div class = 'one-forth w-1/2 border-box'>
+                <label class='flex items-center space-x-2'>
```

Comment:
```suggestion
                <label class="flex items-center space-x-2 w-1/4">
```
looks better, is responsive and uses less code.

additionally, please use an `{#each}` block and create the labels and input like so. this will further increase the readability and shorten the code.